Sierra/challenge_08/loops_challenge.py

```diff
@@ -0,0 +1,109 @@
+
+days = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday']
+
+# QUESTION 1: For loops with list
+
+# Let's start simple, and build up from there.
+# 1.1: Write a for loop that prints out each day in the `days` variable above.
+for day in days:
+    print(day)
+
+
+# 1.2: Now, instead of printing out the day, let's ask the user what their favorite thing
+# to do is on that day of the week. (Make sure to use an f-string so that the user knows which
+# day they're being asked about.)
+
+# for day in days:
+#     input(f'What is your favorite thing to do on {day}? ')
+
+
+
+# 1.3: We should keep track of the user's favorite things to do so that we can print them out all together.
+# ABOVE your for loop, create a new empty list to hold the user's favorite activities.
+fav_activity = []
+
+for day in days:
+    fav_activity.append(input(f'What is your favorite thing to do on {day}? ')) # 1.4
+
+print(fav_activity)
+
+
+# 1.4: Now, back in your for loop, append each of the user's answers into your new list.
+# Print out the list after your loop to check if it got populated correctly.
+
+
+
+# 1.5: After your first loop, let's create a new one. As an example, let's say the user's favorite thing to
+# do on mondays is plan their week. This time, we want the output to be something like this:
+# 'On Mondays, your favorite activity is to plan your week.'
+# We need information from both lists! Let's use the `range` function to loop through the indexes
+# of the items in the lists (this will work because the lists are the same length).
+# Each time through this new loop, use the index number to index into each of your lists for the data
+# you need to print out.
+for i in range(len(days)) and range(len(fav_activity)):
+    print(f'On {days[i]}s, your favorite activity is to {fav_activity[i]}.')
```

Comment:
Not a big deal but the range(len(fav_activity)) is not necessary because they actually have the same length. You can choose either one.